jwst/wavecorr/wavecorr_step.py

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env python
+
+from ..stpipe import Step
+from .. import datamodels
+from . import wavecorr
+
+__all__ = ["WavecorrStep"]
+
+
+class WavecorrStep(Step):
+    """
+    This step applies wavelength offsets to off-center NIRSpec sources.
+    """
+
+    spec = """
+    """
+
+    reference_file_types = ['wavecorr']
+
+    def process(self, input, *args, **kwargs):
```

Comment:
Two point: `process` never gets `kwargs` and the specification should be removed.

Also, unless `process` is intended to get a list, it should only define the argument(s) it needs. Presuming if this case is `input`, then it should be just `input`.

Also, as has been pointed out by others, `input` is a Python builtin function name and we really should start avoiding using that as a variable.

Reply:
This was a copy-n-paste from the previous incarnation of the function that lived in the `extract_2d` step, so I'll update as suggested.

Reply:
Fixed in latest commit